### ISSUE TITLE
Tome Unlock Item: Death Shroud Cloak

### ISSUE THREAD

**Issue by Senthoi** [WarEmu/WarBugs#14399]
Issues should be unique. Check if someone else reported 
the issue first, and please don't report duplicates.
Make sure you've checked the wiki before you post something ToK related:
https://wiki.returnofreckoning.com/

**Expected behavior and actual behavior:**

Collect 5 Broken Wight Blades to complete unlock to allow you to purchase the Death Shroud Cape from the Tome Accessory Librarian in the Lyceum.

Have collected the blades from High Pass. Not able to purchase this cloak, and the Tome Unlock isn't registering either. (Broken Men, Broken Blades) https://wiki.returnofreckoning.com/index.php/Broken_Men,_Broken_Blades
![Volyar_054](https://user-images.githubusercontent.com/22262165/59302829-c4687180-8c8c-11e9-8e10-1f13907e3a4b.jpg)

**Steps to reproduce the problem:**

Seems like this was fixed in 2017 but it's playing up again.

**Testing Screenshots/Videos/Evidences (always needed):**

**Comment by Lheana** [comment]
Hm sorry but it seems like you are able to link the instructions of RoR Wiki, but not to read it... Follow the instructions of RoR Wiki and everything will be fine.

**Comment by Senthoi** [comment]
I followed these instructions, when I tried to equip the Wight blade it said that this was not possible to do. I was trying to place it in the pocket item slot. I'll have another go tomorrow.

**Comment by Lheana** [comment]
No, according to your screenshot, you haven't followed these instructions. It is explicitly written that there are two steps for this unlock, you have only done the first one.

**Comment by Senthoi** [comment]
This screenshot was to show you the items and the reward I was trying to obtain. Here is what I had done:
1. Killed skeletons to obtain 5 X Broken Wight Blades
2. Travelled to Lyceum.
3. Tried to equip blades. Nothing happened when I just clicked them, so I tried to drag them to the pocket item slot. This wouldn't work as the stack of 5, nor as one individually.
I'll try again tonight and take some more explicit screenshots for you if this is still happening and report back on how I get on.
